core/server/models/member.js

```diff
@@ -30,6 +30,9 @@ const Member = ghostBookshelf.Model.extend({
         }, {
             key: 'products',
             replacement: 'products.slug'
+        }, {
+            key: 'newsletters',
+            replacement: 'newsletters.slug'
```

Comment:
are we happy to use slug for this (and products) given that it'll always be `default-newsletter`?